profiles/migrations/0005_alter_profile_comp_region.py

```diff
@@ -13,6 +13,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='profile',
             name='comp_region',
-            field=models.CharField(choices=[('Kyiv', 'Київська область'), ('Vinnytsia', 'Вінницька область'), ('Volyn', 'Волинська область'), ('Dnipro', 'Дніпропетровська область'), ('Donetsk', 'Донецька область'), ('Zhytomyr', 'Житомирська область'), ('Zakarpattia', 'Закарпатська область'), ('Zaporizhzhia', 'Запорізька область'), ('IvanoFrankivsk', 'Івано-Франківська область'), ('Kirovohrad', 'Кіровоградська область'), ('Crimea', 'Крим'), ('Luhansk', 'Луганська область'), ('Lviv', 'Львівська область'), ('Mykolaiv', 'Миколаївська область'), ('Odesa', 'Одеська область'), ('Poltava', 'Полтавська область'), ('Rivne', 'Рівненська область'), ('Sumy', 'Сумська область'), ('Ternopil', 'Тернопільська область'), ('Kharkiv', 'Харківська область'), ('Kherson', 'Херсонська область'), ('Khmelnytskyi', 'Хмельницька область'), ('Cherkasy', 'Черкаська область'), ('Chernivtsi', 'Чернівецька область'), ('Chernihiv', 'Чернігівська область')], default=None, max_length=128, null=True),
+            field=models.CharField(choices=[('Kyiv', 'Київ'), ('Vinnytsia region', 'Вінницька область'), ('Volyn region', 'Волинська область'), ('Dnipro region', 'Дніпропетровська область'), ('Donetsk region', 'Донецька область'), ('Zhytomyr region', 'Житомирська область'), ('Zakarpattia region', 'Закарпатська область'), ('Zaporizhzhia region', 'Запорізька область'), ('IvanoFrankivsk region', 'Івано-Франківська область'), ('Kyiv region', 'Київська область'), ('Kirovohrad region', 'Кіровоградська область'), ('Crimea', 'Автономна Республіка Крим'), ('Luhansk region', 'Луганська область'), ('Lviv region', 'Львівська область'), ('Mykolaiv region', 'Миколаївська область'), ('Odesa region', 'Одеська область'), ('Poltava region', 'Полтавська область'), ('Rivne region', 'Рівненська область'), ('Sevastopol', 'Севастополь'), ('Sumy region', 'Сумська область'), ('Ternopil region', 'Тернопільська область'), ('Kharkiv region', 'Харківська область'), ('Kherson region', 'Херсонська область'), ('Khmelnytskyi region', 'Хмельницька область'), ('Cherkasy region', 'Черкаська область'), ('Chernivtsi region', 'Чернівецька область'), ('Chernihiv region', 'Чернігівська область')], default=None, max_length=128, null=True),
```

Comment:
I think it's better to use `choices=Region.choices` instead of duplicating all the values from it.